nomad/intentions/dgraph-alpha-0-grpc-private.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Kind": "service-intentions",
+  "Name": "dgraph-alpha-0-grpc-private",
+  "Sources": [
+    {
+      "Name": "dgraph-zero-0-grpc-private",
+      "Action": "allow",
+      "Precedence": 9,
```

Comment:
We add the precedence and  Type here so that pulumi doesn't always diff. There are the defaults for consul intentions